File: solutions/2955-account-balance-after-rounded-purchase/solution.py
class Solution(object):
    def accountBalanceAfterPurchase(self, purchaseAmount):
        """
        :type purchaseAmount: int
        :rtype: int
        """
        p=purchaseAmount%10;
        if(p>=5):
            purchaseAmount+=(10-p);
        else:
            purchaseAmount-=p;
        return 100-purchaseAmount;

File: solutions/2955-account-balance-after-rounded-purchase/solution.js
/**
 * @param {number} purchaseAmount
 * @return {number}
 */
var accountBalanceAfterPurchase = function(purchaseAmount) {
    var p=purchaseAmount%10;
        if(p>=5)
            purchaseAmount+=(10-p);
        else
            purchaseAmount-=p;
        return 100-purchaseAmount;
};
